addon/ExperimentProvider.js

```diff
@@ -67,6 +81,12 @@ exports.ExperimentProvider = class ExperimentProvider {
       return;
     }
 
+    if (prefService.get(`extensions.${preferencesBranch}.activateExperiments`) === false) {
```

Comment:
I don't think we usually explicitly check against `true` or `false`, so a plain `!` check should be good.